app/views/fragments/checkout/promoCode.scala.html

```diff
@@ -19,7 +19,7 @@
         <div><strong>✓&nbsp;&nbsp;Promotion applied</strong></div>
         <div class="js-promo-code-snippet"></div>
         <div class="u-note">
-             <a class="u-link js-promo-code-tsandcs" href="#" target="_blank">See full terms and conditions</a>
+             <a class="u-link js-promo-code-tsandcs" href="#" target="_blank" rel="noopener noreferrer">See full terms and conditions</a>
```

Comment:
What does this mean/do?

Reply:
without it the window object is available to the calling page.https://mathiasbynens.github.io/rel-noopener/